server/account/account.go

```diff
@@ -13,8 +17,65 @@ const (
 
 type AccountID [HashSize]byte
 
-func New(pk [pki.PubKeySize]byte) AccountID {
+// NewID generates a unique account id with the provided public key bytes.
+func NewID(pk [pki.PubKeySize]byte) AccountID {
 	// hash the pubkey hash(hash(pubkey))
 	h := HashFunc(pk[:])
 	return HashFunc(h[:])
 }
+
+// Account represents a dex account.
+type Account struct {
+	ID      AccountID
+	PrivKey *secp256k1.PrivateKey
+}
+
+// NewAccount creates a new dex account. The associated private key of the
+// account is generated randomly.
+func NewAccount() (*Account, error) {
```

Comment:
Depending on how you decided to rename `Account` in response to buck's comment, you may ignore the following comment.  Since this is the `account` package, `account.New` is a sufficient syntax. Other constructors can be spelled out though.